### ISSUE TITLE
Custom Miner Download On Bleeding Edge / Vega

### ISSUE THREAD

**Issue by MaynardMiner** [minershive/hiveos-linux#84]
![image](https://user-images.githubusercontent.com/38481606/57031742-83555a00-6c16-11e9-96ee-6af39eab4734.png)

It seems to be an SSL error. I figured out if I add:

```
if [ -f /usr/lib/x86_64-linux-gnu/libcurl-compat.so.3.0.0 ]; then
    echo "Exporting Libcurl"
    export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
fi
```

It fixed on the custom miner h-run.sh side, and wget works. But users can't seem to get to the point to download a custom miner first.

**Comment by minershive** [comment]
Please run wget with verbose mode to understand the problem.

**Comment by MaynardMiner** [comment]
I explained to first user that was helpful to assist me in troubleshooting how to add -v to wget on custom-get script.

wget works fine normally, it is only in miner screen does it have an SSL issue.

I will post his reply, if not, I will test myself later on this afternoon and reply.

**Comment by MaynardMiner** [comment]
![image](https://user-images.githubusercontent.com/38481606/57110581-00f89300-6d07-11e9-82eb-4727f1412862.png)

![image](https://user-images.githubusercontent.com/38481606/57110611-140b6300-6d07-11e9-802b-d171f340868d.png)


He added the -v to wget in custom-get, he received the same output.

What I can tell you, is that it did the same thing for SWARM when SWARM self-downloads powershell, and when I added the above code to SWARM's h-run.sh wget now works, but only after SWARM has started. (LD_LIBRARY_PATH) is set.

It may just be as simple as adding the above code I posted to custom-get. I just don't know if it breaks other miners by doing so.
